src/Documentation/2.0/Migration1.5.md

```diff
@@ -12,3 +12,53 @@ The bulk of the Orleans APIs stayed unchanged in 2.0 or implementation of those
+
+In 1.5, logging configuration is done through `ClientConfiguration` and `NodeConfiguration`. You can configure `DefaultTraceLevel`, `TraceFileName`, `TraceFilePattern`, `TraceLevelOverrides`, `TraceToConsole`, `BulkMessageLimit`, `LogConsumers`, etc through it. In 2.0, logging configuration is consistent with ASP.NET Core 2.0 logging, which means most of the configuration is done through `Microsoft.Extensions.Logging.ILoggingBuilder`. 
+
+To configure `DefaultTraceLevel` and `TraceLevelOverrides`, you need to apply [log filtering](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging) to `ILoggingBuilder`. For example, to set trace level to 'Information' on your grain class `MyGrain`, you can use sample below, 
+```
+siloBuilder.AddLogging(builder=>builder.AddFilter(typeof(MyGrain).FullName, LogLevel.Information));
+```
+If you want to set a default minimum trace level to be Debug, use sample below
+```
+siloBuilder.AddLogging(builder=>builder.SetMinimumLevel(LogLevel.Debug);
+```
+For more information on log filtering, please see their docs on https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging;
+
+To configure TraceToConsole to be `true`, you need to add a `Microsoft.Extensions.Logging.Console.ConsoleLoggerProvider` to `ILoggingBuilder`. The same with `TraceFileName` and `TraceFilePattern`, if you want to log messages to a file, you need to add a `FileLoggerProvider` to `ILoggingBuilder` and configure the file name within `FileLoggerProvider`. Unfortuantely ASP.NET didn't provide a native `FileLoggerProvider` implementation yet, so you can eithter use a third-party file logger provider, or use our implementation of `FileLoggerProvider`, which preserve old logging format of orleans, or you can write your own implementation. 
+
+If you still want to use Message Bulking feature, You need to configure it through `ILoggingBuilder` as well. Message bulking feature lives in our `EventBulkingLoggerProvider` in `Orleans.Logging.Legacy` package. So you need to add dependency on that package first. And then configure it through `ILoggingBuilder` using the extension method below:
+```
+        public static ILoggingBuilder AddMessageBulkingLoggerProvider<TDecoratedLoggerProvider>(this ILoggingBuilder builder, TDecoratedLoggerProvider provider, EventBulkingOptions eventBulkingOptions = null)
+            where TDecoratedLoggerProvider : ILoggerProvider
+```
+This method would apply message bulking feature to any `TDecoratedLoggerProvider` you choose.
+			
+If you still want to use `ILogConsumer`, with message bulking feaure alone side, you need to configure it through `ILoggingBuilder` as well. The support for `ILogConsumer` lives in `Orleans.Logging.Legacy` package. So you need to add dependency on that package first, and then configure Log consumers through extension method below: 
```

Comment:
I would focus on ILogConsumer here, not necessarily bulking (that's just an option, but not the key of why would someone choose ILogConsumer). I would mention that this is useful mainly if you invested in custom implementations of ILogConsumer and cannot convert them to implementations of ILoggerProvider